frontend/scenes/GameScene.js

```diff
@@ -14,68 +14,58 @@ export default class GameScene extends Phaser.Scene {
   }
 
   preload() {
-    this.fetchImages();
     retrieveSoundEffects().forEach((sound) => {
       this.load.audio(sound.name, sound.url);
     });
     this.load.image("bg", "/assets/background.jpg");
+    this.fetchImages()
+        .then(() => {
+          this.load.start();
```

Comment:
why is this needed besides all the load.image(...)?

Reply:
load.image / audio etc. don't actually load the objects, it puts them in a load queue (preload). load.start() actually loads them. Without load.start, I believe the queue starts to be loaded when the create function is loaded, but then it means not every item might be loaded when the cards etc are created. So you need to ensure load.once("complete", fn) to make sure that every item in the queue has been loaded. 
It is not fully intuitive but the load complete doesn't work without the load.start.